pkg/kubeconfig/kubeconfig.go

```diff
@@ -150,6 +146,17 @@ func ForClusterTokenBased(cluster *managementv3.Cluster, clusterID, host, token
 			}
 			nodes = append(nodes, clusterNode)
 		}
+	} else {
+		for _, v3Node := range cluster.NodesFromCAPI {
+			if v3Node.ControlPlane {
+				nodes = append(nodes, node{
+					ClusterName: v3Node.RequestedHostname,
```

Comment:
Is the name format being changed on purpose? I would think it should be same as above to keep the kubeconfig consistent.

Reply:
Yes, the `RequestedHostname` already has the cluster name in it. For example, `donnie-test-cluster-nodepool-pool1-d82r8`.